rotkehlchen/history/price.py

```diff
@@ -159,6 +196,17 @@ def query_historical_price(
         if from_asset == to_asset:
             return Price(FVal('1'))
 
+        special_asset_price = None
+        if from_asset == A_KFEE:
+            special_asset_price = PriceHistorian().get_price_for_special_asset(
+                from_asset=A_KFEE,
+                to_asset=to_asset,
+                timestamp=timestamp,
+            )
```

Comment:
you don't need this if check. You already do that inside `get_price_for_special_asset`.


```suggestion
        special_asset_price = PriceHistorian().get_price_for_special_asset(
            from_asset=A_KFEE,
            to_asset=to_asset,
            timestamp=timestamp,
        )
```

Reply:
Altough I make the same check in the `get_price_for_special_asset` I intentionally wrote it in both places to prevent someone from calling it with any other asset

Reply:
That's not needed. Please remove this from here. Even if it's called with other assets it will return `None`.